src/sentry/conf/server.py

```diff
@@ -2421,6 +2421,24 @@ def custom_parameter_sort(parameter: dict) -> tuple[str, int]:
             "platform": "linux/amd64",
         }
     ),
+    "taskbroker": lambda settings, options: (
+        {
+            "image": "ghcr.io/getsentry/taskbroker:latest",
+            "ports": {"50051/tcp": 50051},
+            "environment": {
+                "TASKBROKER_KAFKA_CLUSTER": (
+                    "kafka-kafka-1"
+                    if os.environ.get("USE_NEW_DEVSERVICES") == "1"
+                    else "sentry_kafka"
```

Comment:
hmmm this is a little strange -- I don't expect these two worlds to overlap so I'm surprised we're bothering to adjust the host here